frontend/components/panels/DesktopClassPanel.js

```diff
@@ -73,6 +90,18 @@ export default class DesktopClassPanel extends BaseClassPanel {
     return false;
   }
 
+
+  // handles the sign up for notifications button being clicked,
+  // to allow for user to choose which sections to sign up for
+  handleClick() {
+    // fishy...
+    this.state.showNotificationBoxes = true;
```

Comment:
lets clean this up :) lemme know if ya want me to take a look! :)

Reply:
will try to figure out... don't know why it was angry last time, might hcnage now?